src/hotspot/share/oops/oop.hpp

```diff
@@ -291,7 +291,7 @@ class oopDesc {
   inline static bool is_instanceof_or_null(oop obj, Klass* klass);
 
   // identity hash; returns the identity hash key (computes it if necessary)
-  inline bool has_no_hash();
+  inline bool fast_no_hash_check();
```

Comment:
Seems like `fast_no_hash_check` ought to be later in this grouping.  The preceding comment is about `identity_hash`.

Reply:
Ok, this makes sense.  Thanks.